packages/core/halo/keyring/src/keyring.ts

```diff
@@ -135,6 +135,42 @@ export class Keyring implements Signer {
     }
     return keys;
   }
+
+  /**
+   * A way to import a key pair (for testing purposes only).
+   */
+  async _importKeyPair(privateKeyData: JsonWebKey, publicKeyData: JsonWebKey): Promise<PublicKey> {
```

Comment:
change this to `async impotKey(keyPair: CryptoKeyPair)` this way it could be used for other key types, not just jwk

Reply:
hold on, is this used anywhere?

Reply:
ok, converted to common import func